config/samples/lightgbm-dist/main.py

```diff
@@ -0,0 +1,81 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import logging
+import argparse
+
+from train import train
+
+# from predict import predict
```

Comment:
Is this still needed?

Reply:
Cleaned [here](https://github.com/kubeflow/xgboost-operator/pull/103/commits/56cbbfe81116dc3ee14f347226c242e2cb1cae7a)